site/using.md

```diff
@@ -151,6 +151,28 @@ default:deployment/helloworld  helloworld  quay.io/weaveworks/helloworld:master-
 
 We can see that the controller is no longer automated.
 
+# Rolling back a Controller
+
+Rolling back can be achieved by combining:
+
+- [`deautomate`](#turning-off-automation) to prevent Flux from automatically updating to newer versions, and
+- [`release`](#releasing-a-controller) to deploy the version you want to roll back to.
+
+```sh
+$ fluxctl deautomate --controller=default:deployment/helloworld
+Commit pushed: c07f317
+CONTROLLER                     STATUS   UPDATES
+default:deployment/helloworld  success
+
+$ fluxctl release --controller=default:deployment/helloworld --update-image=quay.io/weaveworks/helloworld:master-a000001
```

Comment:
Might be worth also putting `fluxctl list-images` in here, to show which images are available to be released.

Reply:
Sure, maybe even `list-images` before and after, to make it easier to visualise the impact?

Reply:
Done.